lib/db.js

```diff
@@ -384,6 +397,27 @@ module.exports = function (
     )
   }
 
+  DB.prototype.updateSessionTokenInBackground = function (token, userAgentString) {
+    log.trace({ op: 'DB.updateSessionTokenInBackground', uid: token && token.uid })
+
+    var self = this
+    var d = P.defer()
+
+    // Even though pool.post is async, we still defer calling it
+    // because userAgent blocks
+    setImmediate(function () {
```

Comment:
You're not missing anything, I guess I was overthinking it.

Because this function is always invoked fire-and-forget, Ryan suggested adding the `...InBackground` suffix. As a follow-on from that I just thought, if we always want to respond early, let's respond even earlier.

But yeah, it's unnecessary. I'll remove it.